app/models/spotlight/exhibit.rb

```diff
@@ -95,6 +95,11 @@ def set_default_thumbnail
       self.thumbnail ||= searches.first.try(:thumbnail)
     end
 
+    def reindex_progress
+      return unless resources
+      @reindex_progress ||= ReindexProgress.new(resources.order('updated_at'))
```

Comment:
Not sure if there is a more rails-y wait to do this or if we should add this as an explicit scope.